dataactbroker/check_status.yml

```diff
@@ -0,0 +1,127 @@
+swagger: '2.0'
```

Comment:
Seems kind of pointless to do the swagger if we're just throwing it out, especially if we can't get it to do examples.